callback.py

```diff
@@ -38,7 +38,7 @@
 AppSettings(prefix=prefix)
 if prefix not in ('', 'dev-'):
     AppSettings.logger.critical(f"Unexpected prefix: '{prefix}' — expected '' or 'dev-'")
-door43_stats_prefix = f"door43.{'dev' if prefix else 'prod'}"
+door43_stats_prefix = f"door43-catalog.{'dev' if prefix else 'prod'}"
 job_handler_stats_prefix = f"{door43_stats_prefix}.job-handler" # Can't add .callback here coz we also have .total
```

Comment:
I'm not certain if I should rename this to `door43-catalog....` or `...catalog-job-handler`.